pacientes/urls.py

```diff
@@ -36,6 +36,7 @@
         CarpetaFamiliarCreateView.as_view(),
         name="pacientes.carpeta-familiar.crear",
     ),
+    path(r'^buscar-paciente/(?P<dni>.*)', BuscarPaciente, name="BuscarPaciente"),
```

Comment:
Acá al usar `path` ya no hace falta la expresión regular. [Ejemplo](https://docs.djangoproject.com/en/3.0/topics/http/urls/#example)

```suggestion
    path("buscar-paciente/<int:dni>", BuscarPaciente, name="BuscarPaciente"),
    path("paciente/create", PacienteCreatePopup, name="PacienteCreate"),
```